extensions/common/http/jetty-core/src/main/java/org/eclipse/edc/web/jetty/JettyExtension.java

```diff
@@ -28,7 +31,7 @@
 import java.security.NoSuchAlgorithmException;
 import java.security.cert.CertificateException;
 
-@Provides({ WebServer.class, JettyService.class })
+@Provides({ WebServer.class, JettyService.class, WebServiceConfigurer.class })
```

Comment:
Not needed, as `WebServiceConfigurer` is provided with the `@Provider` annotation